src/core/components/layout/components/tiles/tiles.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { HTMLAttributes, ReactNode } from "react"
+import { SerializedStyles } from "@emotion/core"
+import { Props } from "@guardian/src-helpers"
+import {tileGridColumns, tilesGridContainer} from "./styles";
+import {
+	tilesCollapseBelowDesktop, collapseBelowDesktopTiles,
+	tilesCollapseBelowleftCol, collapseBelowLeftColTiles,
+	tilesCollapseBelowTablet,
+	collapseBelowTabletTiles,
+	tilesCollapseBelowWide, collapseBelowWideTiles
+} from "../tiles/styles";
+import {Breakpoint} from "@guardian/src-foundations/mq";
+
+type Columns = 2 | 3 | 4 | 5
+type StackSpace = 1 | 2 | 3 | 4 | 5 | 6 | 9 | 12 | 24
+
+type GridBreakpoint = Extract<
+	Breakpoint,
+	"mobile" | "tablet" | "desktop" | "leftCol" | "wide"
+	>
+
+type CollapseBreakpoint = Extract<
+	GridBreakpoint,
+	"tablet" | "desktop" | "leftCol" | "wide"
+	>
+
+interface TilesProps extends HTMLAttributes<HTMLDivElement>, Props {
+	columns: Columns
+	space?: StackSpace
```

Comment:
I don't think the space prop is being used currently. Maybe we can get rid of it for now